cg/cli/workflow/mip_dna/base.py

```diff
@@ -18,25 +17,35 @@
 from cg.store.utils import case_exists
 
 LOG = logging.getLogger(__name__)
-EMAIL_OPTION = click.option("-e", "--email", help="email to send errors to", type=str)
+EMAIL_OPTION = click.option("-e", "--email", help="Email to send errors to", type=str)
 PRIORITY_OPTION = click.option(
     "-p",
     "--priority",
     default="normal",
     type=click.Choice(["low", "normal", "high"]),
 )
 START_WITH_PROGRAM = click.option(
-    "-sw", "--start-with", help="start mip from this program.", type=str
+    "-sw", "--start-with", help="Start mip from this program.", type=str
 )
 OPTION_DRY = click.option(
-    "-d", "--dry-run", "dry_run", is_flag=True, help="print to console instead of executing"
+    "-d", "--dry-run", "dry_run", is_flag=True, help="Print to console instead of executing"
 )
+OPTION_PANEL_BED = click.option(
+    "-pb",
```

Comment:
Skip this, just confusing when short names have multiple letters. In unix this would mean "use both -p and -b"